packages/eslint-plugin/src/configs/recommended.json

```
@@ -2,7 +2,7 @@
   "extends": "./configs/base.json",
   "rules": {
     "@typescript-eslint/adjacent-overload-signatures": "error",
-    "@typescript-eslint/ban-ts-ignore": "error",
+    "@typescript-eslint/ban-ts-comment": ["error", { "ts-nocheck": false }],
```

default options should be set in rule itself 
```suggestion
    "@typescript-eslint/ban-ts-comment": "error",
```

---

The default options are set in the rule, but @bradzacher recommended that `ts-nocheck` be `true` by default.

So either the recommended behaviour will change slightly (which in this case I don't entirely think is breaking, since it's additive rather than subtractive), or the default value will need to not include `ts-nocheck`.

I'm fine with either :)

---

we should wait for @bradzacher than with this

---

this is an unfortunate problem with the recommended generator. I ran into it in #1318 as well.
When you deprecated a rule, it shouldn't be removed from the `recommended` set. Removing it from `all` is perfect,  but not the `recommended` set.